docs/csharp/tutorials/nullable-reference-types.md

```diff
@@ -4,7 +4,7 @@ description: This advanced tutorial provides an introduction to nullable referen
 ms.date: 02/19/2019
 ms.custom: mvc
 ---
-# Tutorial: Express your design intent more clearly with nullable and non-nullable reference types
+# Tutorial: express your design intent more clearly with nullable and non-nullable reference types
```

Comment:
Even though we don't cover this scenario in our guidelines, I think that in the case of tutorials, I actually prefer if the title after the : starts with uppercase. It seems to be different than the How to: guidance that we were given.